helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -462,14 +461,5 @@ public DryrunWagedRebalancer(ZkBucketDataAccessor zkBucketDataAccessor, String c
         Map<ClusterConfig.GlobalRebalancePreferenceKey, Integer> preferences) {
       super(zkBucketDataAccessor, clusterName, preferences);
     }
-
-    @Override
-    protected Map<String, ResourceAssignment> computeBestPossibleAssignment(
-        ResourceControllerDataProvider clusterData, Map<String, Resource> resourceMap,
-        Set<String> activeNodes, CurrentStateOutput currentStateOutput,
-        RebalanceAlgorithm algorithm) throws HelixRebalanceException {
-      return getBestPossibleAssignment(getAssignmentMetadataStore(), currentStateOutput,
-          resourceMap.keySet());
-    }
```

Comment:
Why we need to remove this?

Reply:
Discussed offline. TLDR, we need to remove this to consider the handleDelayedRebalanceMinActiveReplica changes to the idealState which are not persisted to BEST_POSSIBLE in assignment metadatastore. Will add javadoc to this verifier to clarify the expected behavior.